apps/web-twig-demo/symfony.lock

```diff
@@ -14,8 +14,8 @@
     "lmc/spirit-web-twig-bundle": {
         "version": "1.27.0"
     },
-    "lmc/twigx-bundle": {
-        "version": "3.2.0"
+    "almacareer/twigx-bundle": {
+        "version": "5.0.0"
```

Comment:
oh, we skipped v4? :O

Reply:
Yes, we have somehow forgotten to upgrade.